js/functions.js

```diff
@@ -1,13 +1,15 @@
 
-function menuShow() {
-    var items = document.getElementsByClassName("navbar-item-collapse");
+function menuShow()
+{
+    let items = document.getElementsByClassName("navbar-item-collapse");
 
     for( i=0 ; i < items.length ; i++ ){
         if (items[i].classList.contains("hide")) {
-            items[i].classList.remove("hide")
-        } else {
-            items[i].classList.add("hide");
+            items[i].classList.remove("hide");
```

Comment:
I prefer if/else statements instead of continues. In my opinion, they state it clearer that only one of the add/remove will happen to element.